Libs/MRML/Core/vtkMRMLClipModelsNode.cxx

```diff
@@ -128,5 +150,8 @@ void vtkMRMLClipModelsNode::PrintSelf(ostream& os, vtkIndent indent)
   os << indent << "YellowSliceClipState: " << this->YellowSliceClipState << "\n";
   os << indent << "GreenSliceClipState:  " << this->GreenSliceClipState << "\n";
   os << indent << "RedSliceClipState:    " << this->RedSliceClipState << "\n";
+  os << indent << " clippingMethod=\"" << ((this->ClippingMethod == vtkMRMLClipModelsNode::
+    Straight) ? "Straight" : ((this->ClippingMethod == vtkMRMLClipModelsNode::
+    WholeCells) ? "WholeCells" : "WholeCellsWithBoundary")) << "\n";
```

Comment:
Since this is used here in `writeXML` and in `Copy`, could you put this in a separate function? like `GetClippingMethodName` with a switch on `ClippingMethod` within? (@lassoan could you confirm that's how we usually do?)